Lib/opcode.py

```diff
@@ -295,12 +267,29 @@ def pseudo_op(name, op, real_ops):
 
 MAX_PSEUDO_OPCODE = MIN_PSEUDO_OPCODE + len(_pseudo_ops) - 1
 
-del def_op, name_op, jrel_op, jabs_op, pseudo_op
+del def_op, pseudo_op
 
 opname = ['<%r>' % (op,) for op in range(MAX_PSEUDO_OPCODE + 1)]
 for op, i in opmap.items():
     opname[i] = op
 
+# _opcode may not be ready during early stages of the build
```

Comment:
Is that separate from Larry's limerick at the top?

Reply:
It's the same thing. I'm planning to try and get rid of this issue - figure out what is needed in the build and separate it out somehow.

Reply:
Bot we've gotta keep a limeric in. :-)

Reply:
Is it in the stable ABI?